src/components/figures/chart/scorecard/chart_scorecard.ts

```diff
@@ -120,7 +120,7 @@ export class ScorecardChart extends Component<Props, SpreadsheetChildEnv> {
   }
 
   get secondaryFontColor() {
-    return relativeLuminance(this.primaryFontColor) <= 0.3 ? "#757575" : "#bbbbbb";
+    return relativeLuminance(this.primaryFontColor) <= 0.3 ? "#525252" : "#C8C8C8";
```

Comment:
The rule is not that clear to understand and does not make much sense. `primaryFontColor` is the runtime.fontColor, which  is computed by `chartFontColor` which spits out back or white and nothing in between.
 The luminance rule here is supposed to mimic the behaviour of  `chartFontColor`  and thus should be applied to  `runtime.background`.
```suggestion
    return relativeLuminance(this.backgroundColor) > 0.3 ? "#525252" : "#C8C8C8";
```

Reply:
Thanks for pointing that out. I have made changes.